plugins/source/digitalocean/codegen/resources.go

```diff
@@ -0,0 +1,373 @@
+package main
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/s3/types"
+	"github.com/cloudquery/cloudquery/plugins/source/digitalocean/codegen/recipes"
+	"github.com/cloudquery/cloudquery/plugins/source/digitalocean/resources/services/spaces"
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/digitalocean/godo"
+)
+
+var Resources = []*recipes.Resource{
+	{
+		Service: "account",
+		ExtraColumns: []codegen.ColumnDefinition{
+			{
+				Name:     "uuid",
+				Type:     schema.TypeString,
+				Resolver: `schema.PathResolver("UUID")`,
+				Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+			},
+		},
+		SkipFields: []string{"UUID"},
+	},
+	{
+		Service: "cdn",
```

Comment:
I think we may want to make this plural too, for consistency, i.e. cdns, but I can see how that's not super easy to read in this case, so I'll let you decide 😄

Reply:
I saw there are some other tables like that already, like `vpns`, so we should probably make this `cdns`.